signbank/settings/base.py

```diff
@@ -239,3 +239,5 @@
 DEFAULT_ALLOWED_HOSTS = ['127.0.0.1', 'localhost']
 ALLOWED_HOSTS = os.environ.get("ALLOWED_HOSTS").split(
     ",") if os.getenv("ALLOWED_HOSTS") else DEFAULT_ALLOWED_HOSTS
+
+X_FRAME_OPTIONS = 'SAMEORIGIN'
```

Comment:
Can you please add a comment to mention that we need to specify this because django-summernote, the rich text editor used for flatpage editing, embeds itself into the page using an iframe.

Reply:
I literally took your advice :smile: